lib/cadet_web/controllers/auth_controller.ex

```diff
@@ -56,24 +56,35 @@ defmodule CadetWeb.AuthController do
     send_resp(conn, :bad_request, "Missing parameter")
   end
 
+  @doc """
+  Receives a /refresh request with valid attribute.
+
+  Exchanges the refresh_token with a new access_token.
+  """
   def refresh(conn, %{"refresh_token" => refresh_token}) do
     case Guardian.exchange(refresh_token, "refresh", "access") do
-      {:ok, {refresh_token, _}, {access_token, _}} ->
-        render(conn, "token.json", access_token: access_token, refresh_token: refresh_token)
+      {:ok, {_refresh_token, _}, {access_token, _}} ->
+        render(conn, "access_token.json", access_token: access_token)
```

Comment:
We're not giving them a new refresh token (sliding window behaviour)?

Reply:
Oh, hmm okay I realise why we had this now. Will revert back to the old one then.

Also need to change from `exchange` to `encode_and_sign` then, as in totally issuing a new one. Will make the necessary changes.